ui/src/components/TrafficPage.tsx

```diff
@@ -58,6 +59,9 @@ export const TrafficPage: React.FC<TrafficPageProps> = ({onTLSDetected, setAnaly
     const [noMoreDataTop, setNoMoreDataTop] = useState(false);
     const [isSnappedToBottom, setIsSnappedToBottom] = useState(true);
 
+    const [serviceMapStatus, setServiceMapStatus] = useState(false);
```

Comment:
is serviceMapStatus still needed?

Reply:
no, I will remove it, we can use the window["isServiceMapEnabled"] instead